sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 5.1.0-beta.8 (Unreleased)
 
+### Breaking Changes
+- Renamed type `PiiEntityDomainType` to `PiiEntityDomain`.
+- Renamed property name `result` to `documentsResults` in `AnalyzeSentimentActionResult`, `ExtractKeyPhrasesActionResult`,
```

Comment:
although here you say rename to `documentsResults` all your changes are using `document` in singular.
If you want to match what we did in .NET, then it should be `documents`. See https://apiview.dev/Assemblies/Review/b196a5e53fa2441f92e3b1accefa35e7/2f4170304b4f4269aa3e0f6e1853b3d8?doc=False#Azure.AI.TextAnalytics.AnalyzeSentimentActionResult

Reply:
address it in https://github.com/Azure/azure-sdk-for-java/pull/22329